package/src/renderer/__tests__/Shader.spec.tsx

```diff
@@ -113,4 +119,40 @@ describe("Test Shader component", () => {
     );
     processResult(surface, "snapshots/runtime-effects/spiral.png");
   });
+
+  it("should blend cyan/magenta/yellow to black (multiply)", () => {
+    const { vec } = importSkia();
+    const r = width / 2;
+    const c = vec(r, r);
+    const surface = drawOnNode(
+      <Fill>
+        <Blend mode="multiply">
+          <RadialGradient r={r} c={c} colors={["cyan", "magenta"]} />
+          <RadialGradient r={r} c={c} colors={["magenta", "yellow"]} />
+          <RadialGradient r={r} c={c} colors={["yellow", "cyan"]} />
+        </Blend>
+      </Fill>
+    );
+    processResult(surface, "snapshots/runtime-effects/blend-multiply.png");
+  });
+
+  it("should blend more than two shader children 2", () => {
```

Comment:
@naraB I'm trying to use Figma to check the reference test results but I'm not 100% yet that the feature is correct 😅

Reply:
maybe we use just one color within the gradients, i.e. `['green', 'green'], ['yellow', 'yellow']` and run similar tests to the `blendMode` property

or are those two different?